sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/XmlSensorTest.java

```diff
@@ -169,4 +215,28 @@ public void should_analyze_file_with_its_own_encoding() throws IOException {
     assertThat(context.measure(componentKey, CoreMetrics.LINES).value()).isEqualTo(2);
   }
 
+  private void assertLog(String expected, boolean isRegexp) {
```

Comment:
@pynicolas I originally planed to invoke the method with isRegxp = false. I ended up not doing so. But the case isRegexp is simple and available.